src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -0,0 +1,182 @@
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceTaxesPageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.TAXES>;
+
+function WorkspaceTaxesPage({policy, route}: WorkspaceTaxesPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [selectedTaxesIDs, setSelectedTaxesIDs] = useState<string[]>([]);
+    const defaultExternalID = policy?.taxRates?.defaultExternalID;
+    const foreignTaxDefault = policy?.taxRates?.foreignTaxDefault;
+
+    const textForDefault = useCallback(
+        (taxID: string): string => {
+            if (taxID === defaultExternalID && taxID === foreignTaxDefault) {
+                return translate('common.default');
+            }
+            if (taxID === defaultExternalID) {
+                return translate('workspace.taxes.workspaceDefault');
+            }
+            if (taxID === foreignTaxDefault) {
+                return translate('workspace.taxes.foreignDefault');
+            }
+            return '';
+        },
+        [defaultExternalID, foreignTaxDefault, translate],
+    );
+
+    const taxesList = useMemo<ListItem[]>(
+        () =>
+            Object.entries(policy?.taxRates?.taxes ?? {})
+                .map(([key, value]) => ({
+                    text: value.name,
+                    alternateText: textForDefault(key),
+                    keyForList: key,
+                    isSelected: !!selectedTaxesIDs.includes(key),
+                    isSelectable: key !== defaultExternalID,
+                    rightElement: (
+                        <View style={styles.flexRow}>
+                            <Text style={[styles.disabledText, styles.alignSelfCenter]}>
+                                {value.isDisabled ? translate('workspace.common.disabled') : translate('workspace.common.enabled')}
+                            </Text>
+                            <View style={[styles.p1, styles.pl2]}>
+                                <Icon
+                                    src={Expensicons.ArrowRight}
+                                    fill={theme.icon}
+                                />
+                            </View>
+                        </View>
+                    ),
+                }))
+                .sort((a, b) => a.text.localeCompare(b.text)),
+        [policy?.taxRates?.taxes, textForDefault, defaultExternalID, selectedTaxesIDs, styles, theme.icon, translate],
+    );
+
+    const isLoading = taxesList === undefined;
```

Comment:
@filip-solecki Why don't you add `!isOffline` condition as we did in the category and tag page

Reply:
Added!